### PR TITLE
Formatting and Configuration Updates

### PR DESCRIPTION
**Pull request by grkvlt**
Updated Kubernetes to version 1.4.3, quote labels consistently, and fix configuration for provisioning properties keys `minRam` and `minCores`. Also splits up the `src/main/resources` folders more logically.

### CODE REVIEW

File: kubernetes/pom.xml [hunk 1]
```diff
@@ -17,12 +17,17 @@
...
+                <filtering>false</filtering>
+            </resource>
+            <resource>
                 <directory>src/main/resources</directory>
                 <filtering>false</filtering>
             </resource>
             <resource>
                 <directory>src/main/examples</directory>
                 <filtering>false</filtering>
+                <targetPath>examples</targetPath>
```
Comment (m4rkmckenna) [inline kubernetes/pom.xml]: Unsure about packaging examples ... 
I think its useful for documentation but would you ever deploy the examples as is?

Comment (grkvlt) [inline kubernetes/pom.xml]: Correct. Have removed these from the Jars

File: common/pom.xml [hunk 2]
```diff
@@ -46,17 +46,22 @@
                         <configuration>
                             <artifacts>
                                 <artifact>
-                                    <file>${project.basedir}/src/main/resources/docker/docker.bom</file>
+                                    <file>catalog/docker/docker.bom</file>
```
Comment (m4rkmckenna) [inline common/pom.xml]: Not sure if we still need `${project.basedir}` I know this was needed when using maven with `-f`

Comment (grkvlt) [inline common/pom.xml]: Good point, I think that's right. Will keep it there to be sure, then.

**Pull request merged**